recipe/meta.yaml

```diff
@@ -1,50 +1,50 @@
-{% set version = "2.3" %}
+{% set version = "2.3.0" %}
 
 package:
   name: libhdfs3
   version: "{{ version }}"
 
 source:
-#  fn: libhdfs3-{{ version }}.tar.gz
-#  url: https://github.com/Pivotal-Data-Attic/pivotalrd-libhdfs3/archive/v{{ version }}.tar.gz
-#  sha256: b68d112a3c234a50f646071b1de07e5e191018f6762492cd04d57390e1fd5a16
-  git_url: https://github.com/martindurant/libhdfs3-downstream.git
```

Comment:
I don't really know the canonical source for this so I'm use @martindurant's fork that was here before.